projects/packages/my-jetpack/src/products/class-search.php

```diff
@@ -86,4 +87,17 @@ public static function get_pricing_for_ui() {
 			'promotion_percentage' => '50',
 		);
 	}
+
+	/**
+	 * Checks whether the current plan of the site already supports the product
+	 *
+	 * Returns true if it supports. Return false if a purchase is still required.
+	 *
+	 * Free products will always return true.
+	 *
+	 * @return boolean
+	 */
+	public static function has_required_plan() {
+		return ( new Search_PLan() )->supports_search();
```

Comment:
Is `Search_PLan` a typo? Looks like it ought to be `Search_Plan` 🙂

Reply:
wow, I wonder how it didnt throw any errors